Tensile/Source/client/source/Reference.cpp

```diff
@@ -84,52 +85,48 @@ namespace Tensile
             for(int i = 0; i < batchSize.size(); i++) batchSize[i] = problem.batchSize(i);
             for(int i = 0; i < boundSize.size(); i++) boundSize[i] = problem.boundSize(i);
 
-
-            auto batchCount = CoordCount(batchSize.begin(), batchSize.end());
-            auto freeACount = CoordCount(freeASize.begin(), freeASize.end());
-            auto freeBCount = CoordCount(freeBSize.begin(), freeBSize.end());
+            //auto batchCount = CoordCount(batchSize.begin(), batchSize.end());
+            //auto freeACount = CoordCount(freeASize.begin(), freeASize.end());
+            //auto freeBCount = CoordCount(freeBSize.begin(), freeBSize.end());
             auto boundCount = CoordCount(boundSize.begin()+1, boundSize.end());
 
-#pragma omp parallel for collapse(3)
-            for(size_t batchNum = 0; batchNum < batchCount; batchNum++)
-            for(size_t freeANum = 0; freeANum < freeACount; freeANum++)
-            for(size_t freeBNum = 0; freeBNum < freeBCount; freeBNum++)
-            {
-                std::vector<size_t> aCoord(problem.a().dimensions());
-                std::vector<size_t> bCoord(problem.b().dimensions());
-                std::vector<size_t> cCoord(problem.c().dimensions());
-                std::vector<size_t> dCoord(problem.d().dimensions());
+            auto dCount = d.totalLogicalElements();
+            //for(size_t batchNum = 0; batchNum < batchCount; batchNum++)
+            //for(size_t freeANum = 0; freeANum < freeACount; freeANum++)
+            //for(size_t freeBNum = 0; freeBNum < freeBCount; freeBNum++)
```

Comment:
remove commented-out code?

Reply:
I did, in the cleanup commit: https://github.com/ROCmSoftwarePlatform/Tensile/pull/681/files